### PR TITLE
add minVersion to tls.ini options

### PR DESCRIPTION
**Pull request by analogic**
Changes proposed in this pull request:
- add configurable minVersion to tls socket options

Checklist:
- [x] docs updated
- [x] tests updated
- [x] [Changes](https://github.com/haraka/Haraka/blob/master/Changes.md) updated

### CODE REVIEW

File: docs/plugins/tls.md [hunk 1]
```diff
@@ -97,7 +97,17 @@ A list of allowable ciphers to use. Example:
...
-See also: [Strong SSL Ciphers](http://cipherli.st) and the [SSLlabs Test Page](https://www.ssllabs.com/ssltest/index.html)
+See also: [Mozilla SSL configuration generator](https://ssl-config.mozilla.org/) and the [SSLlabs Test Page](https://www.ssllabs.com/ssltest/index.html)
+
+### minVersion
+
+Specifies minimum allowable TLS protocol version to use. Example:
+
+     minVersion=TLSv1.1 
+
+If not set Haraka will use by default node's tls.DEFAULT_MIN_VERSION constant.
```
Comment (msimerson) [inline docs/plugins/tls.md]:
```suggestion
If unset, node's default tls.DEFAULT_MIN_VERSION constant.
```

File: docs/plugins/tls.md [hunk 2]
```diff
@@ -97,7 +97,17 @@ A list of allowable ciphers to use. Example:
...
+
+(**Node.js 11.4+ required**, for older instances you can use *secureProtocol* settings)
```
Comment (msimerson) [inline docs/plugins/tls.md]: With this being the case, we *could* have the plugin check the running node version and map this that setting. That seems like a nicer solution than documenting the issue and foisting it on the user.

Comment (analogic) [reply docs/plugins/tls.md]: Problem is that minVersion setting is not equal to obsolete secureProtocol, it works in slightly different way...

Comment (msimerson) [reply docs/plugins/tls.md]: fair enough

**Pull request merged**